data/README.md

```diff
@@ -20,9 +20,10 @@ $ python preprocess.py --source-lang de --target-lang en \
 
 # Train the model:
 $ mkdir -p checkpoints/fconv
-$ CUDA_VISIBLE_DEVICES=0 python train.py data-bin/iwslt14.tokenized.de-en \
+$ python train.py data-bin/iwslt14.tokenized.de-en \
```

Comment:
I'd put this back, the settings are only good for a single GPU setup, if someone with 8 GPUs runs this line, the resulting model performance will be less good.

Reply:
I see. I put the flag back and add a line of explanation in the comment to make this intention clearer.